devices/leviton.js

```diff
@@ -91,21 +91,26 @@ module.exports = [
         model: 'ZS057-D0Z',
         vendor: 'Leviton',
         description: 'Wall switch, 0-10V dimmer, 120-277V, Lumina™ RF',
-        extend: extend.light_onoff_brightness({disableEffect: true, noConfigure: true}),
-        fromZigbee: [fz.brightness, fz.identify, fz.lighting_ballast_configuration, fz.level_config],
-        toZigbee: [tz.light_onoff_brightness, tz.ballast_config, tz.level_config],
+        // Do not use extend.light_onoff_brightness because that has fz.ignore_basic_report,
+        // which is what tells us when the physical switch is used
```

Comment:
> which is what tells us when the physical switch is used

I would expect that a new converter is added which publishes actions when something is published to the basic cluster.

Reply:
Good point. The `extend:` doesn't seem to be working as I expect. If I keep it, then explicitly add `fz.on_off` things work fine, which is what I did this this seems to match everything else - but feels like I am missing something. Nevertheless, this works locally like this!

Reply:
When I say mostly - really wish I could get some more advanced settings like power on level & min/max on the device itself. I suspect some of these are hidden behind these manufacturer-specific clusters (32776, the next one which is obviously wrong)?

<img width="352" alt="image" src="https://user-images.githubusercontent.com/1923186/200180245-2254ce9c-4ff8-40bb-ad5e-8abff2296510.png">

Reply:
>  I suspect some of these are hidden behind these manufacturer-specific clusters (32776,

If they are, you need to figure out how it works by sniffing the traffic with the original gateway: https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#with-cc2531

Reply:
Nice, I have a CC2652 so I might have to pick up a CC2531 and give it a shot. In the meantime, I think this improvement is ready to merge and if I get around to it, I'll do some more sniffing :)